sunpy/lightcurve/sources/lyra.py

```diff
@@ -40,12 +40,18 @@ class LYRALightCurve(LightCurve):
 
     PROBA2 was launched on 2 November 2009.
 
+    This class can download and hold either Level 2 data (the default) which
+    has sub-second resolution or Level 3 data which has one minute resolution.
```

Comment:
To be pedantic, I would say "Level 3 which is one-minute averaged data derived from Level 2." rather than "Level 3 data which has one minute resolution."  Level 3 is derived from Level 2.  It isn't a different observing mode.  But as I say, that's highly pedantic! ;)